roles/insert_dns_records/templates/openshift-cluster.conf.j2

```diff
@@ -2,7 +2,13 @@ domain={{ domain }}
 {% if write_dnsmasq_config %}
 domain-needed
 bogus-priv
-listen-address=127.0.0.1,{{ listen_address }}
+listen-address="{{ listen_addresses | join(',') }}"
+{% for  listening_intf in (listening_interfaces | default([])) %}
+interface={{ listening_intf }}
```

Comment:
Don't we need also to tell dnsmasq which interface to use for the DHCP range in case we use more than one interface?

Reply:
No from what I can see of the config it is done via disabling DHCP on a given interface `no-dhcp-interfac`. Or are you saying having an interface per dhcp range?

Reply:
I meant to have dhcp range per interface